stack/mainTemplate.json

```diff
@@ -1,47 +1,148 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    /* The number of nodes to deploy */

```

Comment:
love the comments!  however, doesnn't that make it invalid JSON?  This is some extended JSON?

Reply:
Yeah ARM templates are an extended JSON format